src/viewport/index.tsx

```diff
@@ -457,6 +463,7 @@ class Viewport extends React.Component<ViewportProps & DefaultProps, ViewportSta
                 false
             )
         );
+        document.addEventListener("keydown", this.handleKeyDown, false);
```

Comment:
I tried to add the keydown event to elements within this component but only adding it to the document seemed to actually work.  If anyone can make this part more elegant that would be great, since this component does already do touch and pointer events in a neater way.

Reply:
this seems fine to me. If you wanted to I guess you could make a `this.documentHandlers = {"keydown" : this.handleKeyDown`. Or are you saying you'd prefer it to be attached to a different element?